packages/core/src/screens/PINEnter.tsx

```diff
@@ -266,6 +269,17 @@ const PINEnter: React.FC<PINEnterProps> = ({ setAuthenticated }) => {
     subTitle: {
       marginBottom: 20,
     },
+    forgotPINText: {
+      textAlign: 'left',
+      color: ColorPallet.brand.primary,
+      fontSize: 20,
+    },
```

Comment:
```suggestion
    forgotPINText: {
      textAlign: 'left',
      color: ColorPallet.brand.link,
      fontSize: 20,
    },
```
Also, this is a nit, but is that `textAlign: 'left'` needed?

Reply:
Looks like `textAlign: 'left'` was unnecessary. Nothing changes after removing it.